src/pygeoprocessing/geoprocessing.py

```diff
@@ -719,11 +721,12 @@ def align_and_resize_raster_stack(
                 '"mask_vector_path": %s', vector_mask_options)
         mask_vector_info = get_vector_info(
             vector_mask_options['mask_vector_path'])
-        mask_vector_sr_wkt = mask_vector_info['projection']
-        if mask_vector_sr_wkt is not None and target_sr_wkt is not None:
+        mask_vector_sr_wkt = mask_vector_info['projection_wkt']
+        if mask_vector_sr_wkt is not None and \
```

Comment:
Should `mask_vector_sr_wkt` be `mask_vector_projection_wkt`?

Reply:
Patched this and all other internal variable names that use `..sr_wkt` to `..projection_wkt`